projects/plugins/jetpack/modules/search.php

```diff
@@ -40,6 +40,7 @@ protected function __construct() {
 	 * Return the instance of the new class.
 	 */
 	public static function instance() {
-		return Classic_Search::initialize( get_current_blog_id() );
+		// Explicitly provide the blog ID, just in case.
+		return Classic_Search::instance( Jetpack_Options::get_option( 'id' ) );
```

Comment:
I guess the code might not be needed on WPCOM at all, but probably better be compatible as well to get the blog_id?

Reply:
While we don't intend for this shim class to be synced or used in WPCOM, I suppose it won't hurt.